### DISCUSSION TITLE
Error while building with make

### DISCUSSION THREAD

**Discussion by missyaria** [cagnulein/qdomyos-zwift#568] [Q&A] [answered]
Hi,
I've been trying to install qdomyos-zwift on my Linux machine.
However I've been encountering issues while attempting to build with make.

Error : 

```
/usr/bin/ld: cannot open output file qdomyos-zwift: Is a directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:1032: qdomyos-zwift] Error 1
```

Logs before the error : https://pastie.io/vkftrb.properties

**Comment by cagnulein** [comment, marked as answer]
Did you run the make command from arc folder? Do you run with sudo? Do you have a folder under src called qdomyos-zwift? If so delete it

**Comment by missyaria** [comment]
I started over and deleted the folder under src and it works, thanks!
